packages/server/src/utilities/websocket.js

```diff
@@ -0,0 +1,9 @@
+const { Server } = require("socket.io")
+
+const io = new Server(4000)
+
+io.on("connection", socket => {
+  socket.on("foo", arg => {
```

Comment:
Seems like this is just some left over test code.

Reply:
Nice catch thanks, you're exactly right.